pkg/xds/topology/outbound_test.go

```diff
@@ -1421,7 +1421,7 @@ var _ = Describe("TrafficRoute", func() {
 						},
 					},
 				},
-				mesh: defaultMeshWithMTLSAndZoneEgress,
+				mesh: defaultMeshWithMTLS,
```

Comment:
This is so we test that we add MES clusters even without zone egress setting now